bin/eland_import_hub_model

```diff
@@ -41,6 +41,8 @@ MODEL_HUB_URL = "https://huggingface.co"
 
 
 def get_arg_parser():
+    from eland.ml.pytorch.transformers import SUPPORTED_TASK_TYPES
```

Comment:
Why is this not a top-level import?

Reply:
That's to avoid a kind of https://github.com/elastic/eland/pull/553. I thought `eland.ml.pytorch` must be handled with attention to dependencies. Otherwise, could get like this.

```
$ docker run -it python:3 bash -c "pip install git+https://github.com/elastic/eland.git@main && python -c 'import eland.ml.pytorch'"
...
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.11/site-packages/eland/ml/pytorch/__init__.py", line 18, in <module>
    from eland.ml.pytorch._pytorch_model import PyTorchModel  # noqa: F401
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/eland/ml/pytorch/_pytorch_model.py", line 35, in <module>
    from tqdm.auto import tqdm  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'tqdm'
```

Reply:
Right, I thought `get_arg_parser` was called unconditionally, but in fact it's only called after `assert SUPPORTED_TASK_TYPES`. Which makes sense, thanks.